cc-ai/climatecollection
# Updated with Right Commit History - Base Version of the React App

Made the following changes

1. Using react-styledist to allow for a documentation server. See instructions in README.
2. Added TODO's and uncommented and removed the old code. 
3. Can upload images (*.png) for now and tag them. Makes a dummy API call. 

NOTE: If you upload multiple images, it makes one API call per image right now, can be reconfigured easily. But this might help if you have people uploading large number of large files .


To test:

1. Run the react app.
2. Click upload or drag drop some *.PNG files. 
3. On the next page, see the form being submitted. Verify on your flask backend that you received the files.
4. Add your gcloud API key to change to lat,long and repeat to test that address are converted to lat long
5. Run react-styleguidedist to see the documentation server
